src/DynamicImage/widget/DynamicImage.js

```diff
@@ -152,14 +161,14 @@ mendix.widget.declare('DynamicImage.widget.DynamicImage', {
 			//the microflow
 			if (this.clickmicroflow != '')
 			{
-				mx.processor.xasAction({
+				mx.data.action({
 					error       : function() {
 						logger.error(this.id + "error: XAS error executing microflow");
 					},
 					actionname  : this.clickmicroflow,

```

Comment:
Please check the signature when replace a function by an other.
https://apidocs.mendix.com/7/client/mx.data.html#.action
Note the `actionname` is no function parameter but a object parameter

``` js
mx.data.action({
    params: {
        actionname: "MyFirstModule.PetCat"
    },
    origin: this.mxform,
    callback: function(obj) {
        // no MxObject expected
        alert("Just petted the cat a little");
    },
    error: function(error) {
        alert(error.message);
    },
    onValidation: function(validations) {
        alert("There were " + validation.length + " validation errors");
    }
});
```

Reply:
Fixed